server/game/cards/13-CW/BushidoAdherent.js

```diff
@@ -0,0 +1,26 @@
+const DrawCard = require('../../drawcard.js');
+const { CardTypes } = require('../../Constants');
+const AbilityDsl = require('../../abilitydsl.js');
+
+class BushidoAdherent extends DrawCard {
+    setupCardAbilities() {
+        this.action({
+            title: 'Honor a character',
+            condition: context => context.source.isParticipating(),
+            target: {
+                cardType: CardTypes.Character,
+                cardCondition: card => card.isParticipating(),
+                gameAction: AbilityDsl.actions.honor()
+            },
+            then: {
+                gameAction: AbilityDsl.actions.draw(context => ({ target: context.player.opponent }))
+            },
```

Comment:
Should this really be a then? Card doesn't mention it so multiple would seem more apt.

Reply:
As discussed over PM, gameactions should probably be moved to the gameaction property on the ActionProperties instead.